Makefile

```diff
@@ -23,8 +23,11 @@ generate: operator-sdk
 unittest:
 	go test -count=1 ./cmd/... ./internal/... ./pkg/...
 
-functest:
-	go test -count=1 ./tests/...
+build-functests:
+	./hack/in-container.sh ./hack/build-functests.sh
```

Comment:
Why are the tests built in a container? I did not try to run it, but it seems that the tests binary is not copied from the container, so maybe `make functest` will not work.

Reply:
I still think building tests in a container is unnecessarily complex. I don't know why it is done in `kubevirt` and other operators, maybe it is a good idea.

I leave it to @omeryahud and @ksimon1 to say if it is ok.